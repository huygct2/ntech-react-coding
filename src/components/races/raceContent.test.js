import React from 'react';
import renderer from 'react-test-renderer';
import RaceContent from './raceContent';

test('RaceFooter component', () => {
  const component = renderer.create(
    <RaceContent runners={[{}, {}]} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
