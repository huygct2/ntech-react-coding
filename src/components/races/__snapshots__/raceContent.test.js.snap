// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RaceFooter component 1`] = `
<div
  className="runners has-text-centered"
>
  <div
    className="runners__header columns has-text-left"
  >
    <div
      className="column is-6"
    >
      <span>
        Runner
      </span>
    </div>
    <div
      className="column is-3"
    >
      <span>
        Win
      </span>
    </div>
    <div
      className="column is-3"
    >
      <span>
        Place
      </span>
    </div>
  </div>
  <div
    className="runners__list"
  >
    <div
      className="runners__list__item columns"
    >
      <div
        className="runners__name column is-6 has-text-left"
      >
        <span
          className="has-text-weight-semibold"
        >
          . 
        </span>
      </div>
      <div
        className="column is-3"
      >
        <div
          className="bet-odd box"
        >
          <span>
            0.00
          </span>
        </div>
      </div>
      <div
        className="column is-3"
      >
        <div
          className="bet-odd box"
        >
          <span>
            0.00
          </span>
        </div>
      </div>
    </div>
    <div
      className="runners__list__item columns"
    >
      <div
        className="runners__name column is-6 has-text-left"
      >
        <span
          className="has-text-weight-semibold"
        >
          . 
        </span>
      </div>
      <div
        className="column is-3"
      >
        <div
          className="bet-odd box"
        >
          <span>
            0.00
          </span>
        </div>
      </div>
      <div
        className="column is-3"
      >
        <div
          className="bet-odd box"
        >
          <span>
            0.00
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sports__footer"
  >
    <div
      className="sports__all"
    >
      <a
        className="has-text-weight-bold is-uppercase"
      >
        View full race card
      </a>
    </div>
    <span>
      Prices indicative until login & may change before bet is accepted
    </span>
  </div>
</div>
`;
